documentation/src/asciidoc/setting-up.adoc

```diff
@@ -152,3 +152,82 @@ class CalculatorSpec: Spek({
 IMPORTANT: As mentioned in the <<ide-support>> section, the IDEA plugin won't work if you're using the JUnit 4 runner.
 
 
+
+[[setting-up-android]]
+== Setting up Android
+JUnit Platform does not support android projects out of the box, we need to use https://github.com/mannodermaus/android-junit5.
+
+[source,groovy,subs="attributes"]
+.build.gradle
+----
+buildscript {
+    ...
+
+    // 1. Add the android-junit5 Gradle plugin as a classpath dependency
+    dependencies {
+        classpath "de.mannodermaus.gradle.plugins:android-junit5:${junitPlatformVersion}"
```

Comment:
Maybe explicitly add AGP 3.x?

Reply:
Not everybody uses it, you know. And I don’t know what you meant by that.

Reply:
On a different note: The android-junit5 plugin uses an independent version from the JUnit Platform, so it shouldn't re-use the `$junitPlatformVersion` variable.

Reply:
@ming13 well, rest of this `build.gradle` uses things like new API configurations `testImplementation` that simply won't work on AGP 2.x, so I'd maintain 3.x in the docs

Reply:
@artem-zinnatullin That’s quite a mnemonic-like logic. I would make it explicit.

Reply:
@ming13 ? So we should add AGP 3.x here or I didn't understand you?